tests/gis_tests/distapp/tests.py

```diff
@@ -1,6 +1,6 @@
 from __future__ import unicode_literals
 
-from django.contrib.gis.geos import HAS_GEOS, Point
```

Comment:
Thanks, I committed this change separately so you can rebase your branch so it doesn't appear anymore.